### PR TITLE
Azure Databricks- ARM template for managed services cmk

### PR DESCRIPTION
**Pull request by v-vyedavelly**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/nested_addAccessPolicy.bicep [hunk 1]
```diff
@@ -0,0 +1,24 @@
+param ObjectID string
```
Comment (alex-frankel) [inline quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/nested_addAccessPolicy.bicep:1]: is this file used anywhere?

File: quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
+{
```
Comment (alex-frankel) [inline quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/azuredeploy.json:1]: why is azuredeploy written in JSON vs this other `nested_addAccessPolicy.bicep` written in bicep?

File: quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+description: This template allows you to create an Azure Databricks workspace with managed services and CMK.
+page_type: sample
+products:
+- azure
+- azure-resource-manager
+urlFragment: databricks-workspace-with-managed-svc-customer-managed-keys
+languages:
+- bicep
```
Comment (alex-frankel) [inline quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/README.md:9]: I think if the only file being deployed is JSON, bicep should not be listed as a language in the readme.

**Comment by v-vyedavelly** [comment]
@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by v-vyedavelly** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**